contracts/interfaces/workflows/IRoyaltyWorkflows.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
```

Comment:
Would suggest not using "^" in pragma for security reason. This applies to all contracts in SPG repo.

You may also bump the version to the same version as protocol core

Reply:
Will create a ticket for updating all contracts to use 0.8.26 :)